pyfolio/plotting.py

```diff
@@ -29,6 +29,8 @@
 from . import timeseries
 from . import pos
 
+from .utils import BDAYS_PER_MONTH
```

Comment:
I'm not huge on supporting `timedelta`. If someone really wants to use it, they can always call:
```python
plot_rolling_fama_french(returns, rolling_window=datetime.timedelta(months=3).days)
```

I'm open to renaming to `APPROX_BDAYS_PER_MONTH`.